cmd/branches.go

```diff
@@ -75,14 +76,18 @@ var (
 		Args:  cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
 			ctx := cmd.Context()
+			cmdFlags := cmd.Flags()
 			if len(args) == 0 {
 				if err := promptBranchId(ctx, flags.ProjectRef); err != nil {
 					return err
 				}
 			} else {
 				branchId = args[0]
+				if cmdFlags.Changed("output-env") {
+					output.Value = "env"
+				}
 			}
-			return get.Run(ctx, branchId)
+			return get.Run(ctx, branchId, output.Value, pgconn.Config{}, afero.NewOsFs())
```

Comment:
```suggestion
			return get.Run(ctx, branchId, output.Value, afero.NewOsFs())
```

Also no need to pass in pgconn, as we will be constructing the connection config from branch details returned from api.

Reply:
@sweatybridge , but if we don't pass pgconn here, then it will give an error because in get.Run(), 5 arguments are expected, so are we gonna change the Run() func in `internal/branches/get/get.go`?

Reply:
Yes, let's change the Run method to remove this arg since it's not needed.